addr_linux.go

```diff
@@ -183,7 +183,7 @@ func (h *Handle) AddrList(link Link, family int) ([]Addr, error) {
 	return res, nil
 }
 
-func parseAddr(m []byte) (addr Addr, family, index int, err error) {
+func DeserializeAddr(m []byte) (addr Addr, family, index int, err error) {
```

Comment:
If not required, we should not export methods. I get it `LinkDeserialize` and `NeighDeserialize` made it out, although I am not sure if for good reasons.

Reply:
My thinking behind this was that `nl.SubscribeAt` is already exported, so it's possible for library users to write functionality based on that, which doesn't  actually use the individual `*Subscribe` methods.

This is how I started out subscribing to neighbor events prior to modifying the library itself (this PR), and I still use it in this fashion listening to routes + neighbors + links + addresses from a single `nl.SubscribeAt` call.

In those sorts of implementations, it's nice to have consistent library functions that are able to decode the relevant netlink messages, and as it already existed, I exported it when renaming 😃 

Point taken though that this increases the public API surface area complexity and this may not be the best way forward...